Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/GDSInformation/GDSInformationController.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+/// View controller for `GDSInformation` screen
+///     - `informationImage` (type: `String`)
+///     - `titleLabel` (type: `UILabel`)
+///     - `bodyLabel` (type: `UILabel`)
+///     - `footnoteLabel`  (type: `UILabel`)
+///     - `primaryButton`  (type: ``RoundedButton`` inherits from ``SecondaryButton``)
+///     - `secondaryButton`  (type: ``SecondaryButton`` inherits from ``UIButton``)
+public final class GDSInformationController: BaseViewController, TitledViewController {
```

Comment:
I think this should be `GDSInformationViewController` rather than `GDSInformationController` :) Also the same for the name of the test files.

Reply:
thanks Ben making this change now :)